tests/test_access_request.py

```diff
@@ -560,9 +562,18 @@ def test_resolve_app_access_request_notification(
     assert app_owner_user2 in kwargs['approvers']
     assert user in kwargs['approvers']
 
-def test_auto_resolve_create_access_request(app: Flask, db: SQLAlchemy, okta_group: OktaGroup, user: OktaUser, mocker: MockerFixture) -> None:
+def test_auto_resolve_create_access_request(app: Flask,
```

Comment:
nit: Could this test verify that the `access_request_created` parameters are passed as expected?

Reply:
Added!